### ISSUE TITLE
fly execute randomly can no longer properly relatively resolve inputs

### ISSUE THREAD

**Issue by Freyert** [concourse/fly#228]
I have been using `fly execute` successfully all morning and then all of a sudden this error starts happening 100% of the time when using relative paths.

```
➜  Desktop fly --target ci execute --config `./pipeline/ci/tasks/task_deploy_stack.yaml` --input pipeline-scripts=../../../scripts
executing build 41 at https://34.246.141.101/builds/41
initializing
failed to upload input 'pipeline-scripts': Put https://XXXXX/api/v1/builds/41/plan/5b058e31/input: tar compress failed (exit status 1). output: "tar: ci/tasks/deploy_environment.yaml: Cannot stat: No such file or directory\ntar: Error exit delayed from previous epipeline-scripts: 12.42 MiB/s 0s
Put /volumes/065afdce-adf9-4529-59af-516c96a4db9e/stream-in?path=.: unexpected EOF
errored
```
When I rename the config file to be `./pipeline/ci/tasks/deploy_environment.yaml` it starts working again. 

I guess the issue is. What must I do to rename the config file?

**Comment by antonu17** [comment]
Today I faced the same thing, and I found out that uploading inputs was always failing on removed and not commited files. So it turns out that fly execute is getting list of files to upload from git state instead of real files.

**Comment by TG-KK** [comment]
@antonu17 your point is correct, it works after staging the changes
It's weird that this is required, as **fly execute**'s main purpose was to test the task config locally without commiting _under-development changes_ to source control

**Comment by cirocosta** [comment]
Hey,

Looking at the code (see https://github.com/concourse/fly/blob/9a2f2ebde1136119a17d53057b1b4f273385f3e4/commands/internal/executehelpers/uploads.go#L21-L28), it seems to me that with the `--include-ignored` flag you can get around that, right?

``` 
--include-ignored             Including .gitignored paths. Disregards .gitignore entries and uploads everything
``` 

thx!

**Comment by antonu17** [comment]
Yes, It looks like `--include-ignored` includes not only ignored, but simply all files that are available in current dir, without looking into git stage. :+1: 
Nice catch, @cirocosta :wink: !
